client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts

```diff
@@ -194,6 +221,23 @@ export class MediafileListComponent extends BaseViewComponent implements OnInit,
         this.clearSubscriptions();
     }
 
+    /**
+     * Determine if the given file has any extra option to show.
+     * @param file the file to check
+     * @returns wether the extra menu should be accessible
+     */
+    public hasAnythingToShow(file: ViewMediafile): boolean {
```

Comment:
return this.operator.hasPerms('agenda.can_see_list_of_speakers') || (file.isProjectable() && this.operator.hasPerms('core.can_manage_projector')) || this.canEdit;